telco-hub/configuration/reference-crs-kube-compare/required/acm/acmAgentServiceConfig.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: agent-install.openshift.io/v1beta1
+kind: AgentServiceConfig
+metadata:
+  name: agent
+  annotations:
+    argocd.argoproj.io/sync-wave: "7"
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  databaseStorage:
+    storageClassName: {{ .spec.databaseStorage.storageClassName }}
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: {{ .spec.databaseStorage.resources.requests.storage }}
+  filesystemStorage:
+    storageClassName: {{ .spec.filesystemStorage.storageClassName }}
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: {{ .spec.filesystemStorage.resources.requests.storage }}
+  imageStorage:
+    storageClassName: {{ .spec.imageStorage.storageClassName }}
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: {{ .spec.imageStorage.resources.requests.storage }}
+  mirrorRegistryRef:
+    name: mirror-registry-config
+  {{- if .spec.osImages }}
+  osImages:
```

Comment:
Any content here is acceptable. They are allowed to define what versions they want to make available.

Reply:
Done.